h2o-docs/src/product/data-science/hglm.rst

```diff
@@ -0,0 +1,394 @@
+- The dimension of the random effect coefficients is :math:`q` which can include the intercept. Note that :math:`q \leq p`;
+- :math:`\varepsilon_{ij} \sim N(0, \delta_e^2)`;
+- :math:`u_{ij} \sim N(0, \delta_u^2)`:
+- :math:`\varepsilon_{ij}, u_{mj}` are independent;
+- :math:`u_{mj}, u_{m,j}` are independent if :math:`m \neq m'`.
+
+We need to solve the following parameters: :math:`\beta_{00}, \beta_{0j}, \beta_{m0}, u_{mj}, \delta_e^2, \delta_u^2`.
+
+Defining an HGLM model
+----------------------
+Parameters are optional unless specified as *required*.
+
+Algorithm-specific parameters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- **em_epsilon**: (Only available for EM method) Converge if beta/ubeta/tmat/tauEVar changes less (using L-infinity norm) than EM epsilon (defaults to ``0.001``).
+
+- **gen_syn_data**: If enabled, it will generate synthetic HGLM data with the fixed coefficients specified in ``initial_fixed_effects`` and the random coefficients taken from ``initial_random_effects`` or the random effects are randomly generated. In particular, it will generate the folowing output: :math:`Y_j = A_{fj} \theta_f + A_{rj} \theta_{rj} + r_j`. The gaussian noise is generated with variance that's specified in ``tau_e_var_init``. If the random coefficients are to be randomly generated, they are generated with gaussian distribution with variance that's specified in ``tau_u_var_init``.
+
+- **group_column**: Specify the level-2 variable name which is categorical and used to generate the groups in HGLM (defaults to ``None``).
+
+- **initial_fixed_effects**: An array that contains the initial values of the fixed effects coefficient (defaults to ``None``).
+
+- **initial_random_effects**: An H2OFrame ID that contains the initial values of the random effects coefficient. The row names should be the random coefficient names (defaults to ``None``).
+	
+	.. note::
+
+		If you aren't sure what the random coefficient names are, then build the HGLM model with ``max_iterations=0`` and check out the model output field ``random_coefficient_names``. The number of rows of this frame should be the number of level 2 units. Check out the model output field ``group_column_names``. The number of rows should equal the length of the ``group_column_names``.
+
+- **initial_t_matrix**: An H2OFrame ID that contains the initial values of the T matrix. It should be a positive symmetric matrix (defaults to ``None``).
+
+- **method**: Obtains the fixed and random coefficients as well as the various variances (defaults to ``"em"``).
+
+- `random_columns <algo-params/random_columns.html>`__: An array of random column names from which random effects coefficients will be generated in the model building process.
+
+-  `rand_family <algo-params/rand_family.html>`__: Specify the distribution of the random effects. Currently only ``rand_family=["gaussisan"]`` is supported.
```

Comment:
rand_family is one string, now an array of string.

So, please change 

``rand_family=["gaussisan"]``

to

``rand_family="gaussisan"``